README.md

```diff
@@ -1,145 +1,99 @@
 # Truss
 
-**Serve any model without boilerplate code**
-
-![Truss logo](https://raw.githubusercontent.com/basetenlabs/truss/main/docs/assets/truss_logo_horizontal.png)
+**The simplest way to serve AI/ML models in production**
 
 [![PyPI version](https://badge.fury.io/py/truss.svg)](https://badge.fury.io/py/truss)
 [![ci_status](https://github.com/basetenlabs/truss/actions/workflows/main.yml/badge.svg)](https://github.com/basetenlabs/truss/actions/workflows/main.yml)
 
-Meet Truss, a seamless bridge from model development to model delivery. Truss presents an open-source standard for packaging models built in any framework for sharing and deployment in any environment, local or production.
-
-Get started with the [end-to-end tutorial](https://truss.baseten.co/e2e).
+## Why Truss?
 
-## What can I do with Truss?
+* **Write, deploy, invoke:** Package and test model code, weights, and dependencies with a model server that behaves the same in development and production.
```

Comment:
What happened to write once, run anywhere?

Reply:
Oops, was playing with other tag lines and forgot to put it back. Will put it back.